File: python/003_fizzbuzz/code.py
import sys

sys.argv.pop(0)
limit = int(sys.argv[0]) if sys.argv else 100

for i in range(1, limit + 1):
    string = ''
    string += '' if i % 3 else 'Fizz'
    string += '' if i % 5 else 'Buzz'
    string += str(i) if not string else ''
    print(string)

File: ruby/003_fizzbuzz/code.rb
limit = ARGV.empty? ? 100 : ARGV[0].to_i

limit.times do |i|
  i += 1
  str = ''
  str += 'Fizz' if (i % 3).zero?
  str += 'Buzz' if (i % 5).zero?
  str += i.to_s if str.empty?
  puts str
end
